gunnerus-palfinger-crane/config/OspSystemStructure.xml

```diff
@@ -31,11 +32,11 @@
                     <Real value="0"/>
                 </InitialValue>		
 				<InitialValue variable="vesselZipFile">
-                    <String value="ShipModel.zip"/>
+                    <String value="C:\absolute\path\to\ShipModel.zip"/>
```

Comment:
If we put the zip file inside the FMU (named vesselmodel.zip) we don't need this parameter. I can make the update?